File: android/app/src/main/java/dev/deliteai/assistant/domain/models/HistoryItem.kt
/*
 * SPDX-FileCopyrightText: (C) 2025 DeliteAI Authors
 *
 * SPDX-License-Identifier: Apache-2.0
 */

package ai.nimbleedge.nimbleedge_chatbot.domain.models

import java.util.Date

data class HistoryItem(
    val parentChatId: String,
    val title: String,
    val dateTime: Date
)

File: ios/NimbleEdgeAI/models/HistoryItem.swift
/*
 * SPDX-FileCopyrightText: (C) 2025 DeliteAI Authors
 *
 * SPDX-License-Identifier: Apache-2.0
 */

import Foundation
struct HistoryItem {
    let parentChatId: String
    let title: String
    let dateTime: Date
    let chats: [ChatMessage]
}
